### ISSUE TITLE
Configuration Page Shows Incorrect Changes Count

### ISSUE THREAD

**Issue by Druffib** [SiaFoundation/renterd#1431]
### Current Behavior

When I change a field such as max download price or upload price, this is listed as "2 Changes" near the save changes button.

### Expected Behavior

I would expect changing one field to be "1 Change" near the save changes button.

### Steps to Reproduce

1. Go to the Configuration page
2. Reset all changes in case any are outstanding
3. Change Max Download Siacoin Price
4. Observe count of changes

### Version

v1.0.8

### What operating system did the problem occur on (e.g. Ubuntu 22.04, macOS 12.0, Windows 11)?

Windows 10

### Autopilot Config

N/A

### Bus Config

N/A

### Contract Set Contracts

N/A

### Anything else?

Browser is Firefox 128

**Comment by alexfreska** [comment]
@Druffib hey thank you for reporting this. I think this is likely due to having the allowance "calculate" feature enabled which automatically calculates an ideal value and updates the field when you change your estimates or max prices. 



![Image](https://github.com/user-attachments/assets/2d3a6d42-65d5-40ed-bdbc-6508acd16dae)



This is a confusing user experience and is not clear enough in the UI, but because of feedback like yours we have already removed this behaviour in the next release. In the next version of the software, there is no longer a "calculate" feature that automatically updates the value, instead we have added explicit buttons for settings an optimal allowance or fitting prices to the allowance.
